src/main/java/org/mskcc/oncokb/curation/importer/CdxImporter.java

```diff
@@ -107,7 +107,7 @@ public void importCdxMain() throws IOException {
                             for (String val : columnValue.split("and")) {
                                 Optional<FdaSubmission> fdaSubmissionOptional = parseFdaSubmissionColumn(val);
                                 if (fdaSubmissionOptional.isPresent()) {
-                                    fdaSubmissions.add(fdaSubmissionOptional.get());
+                                    fdaSubmissions.add(fdaSubmissionOptional.orElseThrow());
```

Comment:
No need to check for `isPresent()` if we now use `orElseThrow()`

Reply:
@bprize15 for places if we intend to throw `NoSuchElementException`, we don't need to check. But if the optional is empty and we want to use different logic, then we need to.